packages/esm-patient-test-results-app/src/overview/common-datatable.component.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import {
+  DataTable,
+  Table,
+  TableContainer,
+  TableHead,
+  TableHeader,
+  TableRow,
+  TableCell,
+  TableBody,
+} from 'carbon-components-react';
+import { TypedTableRow } from './helpers';
+import { OverviewPanelData } from '../overview/useOverviewData';
+import styles from './common-datatable.scss';
+
+interface CommonDataTableProps {
+  data: Array<OverviewPanelData>;
+  tableHeaders: Array<{
+    key: string;
+    header: string;
+  }>;
+  title?: string;
+  toolbar?: React.ReactNode;
+  description?: React.ReactNode;
+}
+
+const CommonDataTable: React.FC<CommonDataTableProps> = ({ title, data, description, toolbar, tableHeaders }) => (
+  <DataTable rows={data} headers={tableHeaders} size="short">
+    {({ rows, headers, getHeaderProps, getRowProps, getTableProps, getTableContainerProps }) => (
+      <TableContainer
+        className={styles.tableContainer}
+        title={title}
+        description={description}
+        {...getTableContainerProps()}
+      >
+        {toolbar}
+        <Table {...getTableProps()} isSortable useZebraStyles>
+          <colgroup className={styles.columns}>
+            <col span={1} />
+            <col span={1} />
+            <col span={1} />
+          </colgroup>
```

Comment:
What's this about?

Reply:
Was intended to get control over the DataTable column widths. A Carbon style override would likely do the trick just as well.